INSTALL.md

```diff
@@ -14,7 +14,7 @@ http://www.gtlib.gatech.edu/pub/ubuntu-releases/xenial/
 Or click below for direct download
 http://www.gtlib.gatech.edu/pub/ubuntu-releases/xenial/ubuntu-16.04.4-desktop-amd64.iso
 
-Set up VirtualBox to install from Ubuntu ISO, 4 Gigs of RAM and 20 GB of storage will work.  
+Set up VirtualBox to install from Ubuntu ISO, 4 Gigs of RAM and 20 GB of storage will work great.  For a minimal setup I have also had success with a 2GB RAM and 15GB of storage.  VirtualBox typically will add an equal amount of swap automatically to give you the 4GB RAM that is needed.
```

Comment:
Have you tested making shielded transactions with 2GB of RAM and 2GB of swap? I am not sure that works or it might get super slow.